design/ratelimit-design.md

```diff
@@ -1,175 +1,501 @@
-# Envoy Rate Limiting Support
+# Rate Limiting Support
 
-The rate limit service configuration specifies the global rate limit service Envoy should talk to when it needs to make global rate limit decisions.
-If no rate limit service is configured, a “null” service will be used which will always return OK if called.
+Status: Draft
```

Comment:
Until we make a further policy change about design documents, let's just do this as Status: Accepted.

Reply:
https://github.com/projectcontour/contour/pull/3178/commits/d9beb4da80a3814a026d6e6f578d28f2aac3a2ef